app/views/posts/index.html.erb

```diff
@@ -1,6 +1,31 @@
-<h1>List of all posts</h1>
-<ul>
-  <li>post 1</li>
-  <li>post 2</li>
-  <li>post 3</li>
-</ul>
+<section>
+  <div class="user-item">
+    <div class="image-box">
+      <img src="<% @user.photo %>" alt="user photo">
+    </div>
+    <div class="user-info-box">
+      <h1 class="user-name"><%= @user.name %></h1>
+      <h4 class="posts-count">Number of posts: <%= @user.posts.count %></h4>
+    </div>
+  </div>
+  <ul class="user-posts-list posts-comments">
+    <% @posts.each do |post| %>
+      <li class="posts-comments-box">
+        <%= link_to user_post_path(@user, post) do %>
+        <div class="border user-posts-item">
+          <div>
+            <h4>Post: #<%= post.id %></h4>
+            <p><%= post.text %></p>
+          </div>
+          <p class="comments-likes-count">Comments: <%=post.comments.count %>, Likes: <%= post.likes.count %></p>
+        </div>
+        <% end %>
+        <div class="border comments-box">
+          <% post.comments.each do |comment| %>
+            <p><%= comment.user.name %>: <%= comment.text %></p>
+          <% end %>
```

Comment:
- [ ] Great job styling this page to look like the given design :clap: However, there are still a few changes that need to be made and you'll be on your way to the next Milestone :+1: 
- [ ] I will suggest that just as the previous reviewer mentioned, you make use of the `five_most_recent_comments` method you created during the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md) in the `Post Model`so as to meet the project requirements 😊
- [ ] From the given wireframe, you can see that on the `posts index` page, only the five recent comments are shown, and when on the `posts show` page, then all the comments belonging to a post are displayed. So kindly make that adjustment as well :+1: 

| Your Design | The Given Wireframe |
|--------|--------|
| ![Screenshot from 2023-11-03 14-08-04](https://github.com/Alibaba2023/blog_app/assets/102699958/9e695ca6-6583-4455-8070-0ea348aab4be) | ![image](https://github.com/Alibaba2023/blog_app/assets/102699958/4c53ef99-cf19-4d8c-9be1-5293e58f0e11) | 

- [ ] Also, kindly style the user image properly as I mentioned in my earlier comment so the image isn't in the same container as the `username` and `number of posts` and can look exactly as seen in the given wireframe. KIndly do this for all corresponding pages :wink: